manifests/kustomize/gcp-workload-identity-setup.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Google service Account (GSA)
+SYSTEM_GSA=${SYSTEM_GSA:-$CLUSTER_NAME-kfp-system}
+USER_GSA=${USER_GSA:-$CLUSTER_NAME-kfp-user}
+
+# Kubernetes Service Account (KSA)
+SYSTEM_KSA=(ml-pipeline ml-pipeline-ui)
```

Comment:
Does any other KSA need to be bound to kfp system service account?

Reply:
Do you know why ml-pipeline need gcp access?

Ideally non of these services would require GCP access, for the purpose of secured multi-user architecture. 
/cc @gaoning777

Reply:
I don't know. Let me try removing this.

Reply:
Removed the binding and tests keep passing. I think this is good.